sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/LockRenewalOperation.java

```diff
@@ -96,7 +96,7 @@ class LockRenewalOperation implements AutoCloseable, Disposable {
                 cancellationProcessor.onComplete();
             }, () -> {
                 if (status.compareAndSet(LockRenewalStatus.RUNNING, LockRenewalStatus.COMPLETE)) {
-                    logger.verbose("Renewing session lock task completed.");
+                    logger.verbose("Renewing lock task completed.");
```

Comment:
logger instance will log "isSession:true|false" key-value. This log line is misleading in case of message lock renewal use case.